### DISCUSSION TITLE
Is there a way to run a tilemaker process without running the node_function ?

### DISCUSSION THREAD

**Discussion by BenOnTrack** [systemed/tilemaker#539] [Q&A] [answered]
i'm experiencing an issue on my way_function that I'd like to troubleshoot. Each time i'm running tilemaker with my process file, the script starts by running the node_function - even if this one is emptied, it's still take a bit of a time before reaching the way_function while I'm looking for a quick way to only run the way_function to support my troubleshooting. 

Any way to do so ?

**Comment by systemed** [comment, marked as answer]
No, I'm afraid not - tilemaker always reads nodes first, then ways. If you create a smaller extract, and disable any shapefiles that you're loading, that will mean you get to the ways quicker!

**Reply by BenOnTrack** [reply]
thanks! that's what I thought when reading the code. No worry!
